01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,19 @@
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+  if (req.url === '/') {
+    res.end('Welcome to our home page');
+    return;
+  }
+  if (req.url === '/about') {
+    res.end('Here is our short history');
+    return;
+  }
+  res.end(`
+  <h1>Oops!</h1>
+  <p>We can't seem to find the page you are looking for</p>
+  <a href="/">back home</a>
+  `);
```

Comment:
Nice finding you can 'return early' if one of your `if` clauses is true to prevent further code from executing. 💯